purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -76,6 +76,9 @@ class Purchases internal constructor(
     val appUserID: String
         @Synchronized get() = purchasesOrchestrator.appUserID
 
+    val storefrontCountryCode: String?
+        @Synchronized get() = purchasesOrchestrator.storefrontCountryCode
```

Comment:
"Storefront" is quite an Apple-focused term which I haven't really seen being used in relation to Google Play / Android, although it does cover the meaning well. 

If we want to make this more generic we could opt for `storeCountryCode`? (On both platforms I mean.) What do you think?

Reply:
storefront is used by Apple, but it is also generic term that can be used to refer to the user-facing aspect of an online store, which we've adopted into the Android SDK already.

```kotlin
interface StorefrontProvider {
    fun getStorefront(): String?
}
```

```kotlin
    /**
     * Obtain store country code in ISO 3166-1-alpha-2 standard format.
     * Null if there has been an error.
     */
    abstract fun getStorefront(
        onSuccess: (String) -> Unit,
        onError: PurchasesErrorCallback,
    )
```

Reply:
Yea I'm totally fine with keeping it! It's a word and it has a meaning and it applies here 😄